src/odemis/acq/stream/_sync.py

```diff
@@ -1118,7 +1275,19 @@ def _runAcquisitionEbeam(self, future):
 
                     # integrate/sum images
                     self._integrateImages(integration_count)
-                    logging.debug("Done acquiring image number %s out of %s.", n, tot_num)
+
+                    for i, das in enumerate(self._acq_data):
+                        self._assembleLiveData(i, das[-1], px_idx, rep, pol_idx)
+
+                    # Activate _updateImage thread
+                    self._shouldUpdateImage()
+
+                    # Live update the setting stream with the new data
+                    self._sccd._onNewData(self._ccd_df, self._acq_data[self._ccd_idx][-1])
+
+                    logging.debug("Done acquiring image number %s out of %s." % (n, tot_num))
```

Comment:
Typically, in logging, it's considered nicer to read to have each variable passed as a separate parameter (as it used to be for this line). So no need to change to using `%`.